docs/getting_started/tutorial.rst

```diff
@@ -2,66 +2,128 @@ Your first Wagtail site
 =======================
 
 .. note::
-   This tutorial covers setting up a brand new Wagtail project. If you'd like to add Wagtail to an existing Django project instead, see :doc:`integrating_into_django`.
+   This tutorial covers setting up a brand new Wagtail project.
+   If you'd like to add Wagtail to an existing Django project instead, see :doc:`integrating_into_django`.
 
-1. Install Wagtail and its dependencies:
+Install and run Wagtail
+-----------------------
 
-   .. code-block:: console
+Install dependencies
+~~~~~~~~~~~~~~~~~~~~
+
+Wagtail supports Python 3.4, 3.5, 3.6, and 3.7.
+
+To check whether you have an appropriate version of Python 3:
+
+.. code-block:: console
+
+   $ python3 --version
+
+If this does not return a version number or returns a version before 3.4, you will need to `install Python 3 <https://www.python.org/downloads/>`_.
```

Comment:
before 3.4 -> lower than 3.4

Reply:
Thanks! Another case where the word didn't sound right, but the right one just hadn't come to me :) This is much better.